apps/dsa.c

```diff
@@ -242,14 +240,14 @@ int dsa_main(int argc, char **argv)
     }
 
     if (outformat == FORMAT_ASN1 || outformat == FORMAT_PEM) {
-        if (pubout || pubin)
+        if (pubin || pubout)
```

Comment:
No need to change the order here or immediately below.

Reply:
Reverted both. This was just for consistency with the same statement in related `apps/*.c` files.